config/locales/en/shared.yml

```diff
@@ -4,5 +4,4 @@ en:
     browse:
       prefix: 'Browse:'
       title: Browse GOV.UK
-    language_direction: rtl
```

Comment:
```
language_direction: ltr
```

I think it's better to be explicit, it makes it easier to see all the locale keys we've got using automated tools.